evalai/utils/auth.py

```diff
@@ -20,10 +20,12 @@ def get_user_auth_token_by_login(username, password):
     """
     Returns user auth token by login.
     """
+    from evalai.utils.requests import check_compatibility
```

Comment:
@DeeJayBro can we place this import with all other imports at the start of the file

Reply:
Nope, because it would lead to circular imports